### DISCUSSION TITLE
Two visualization questions

### DISCUSSION THREAD

**Discussion by irfanimaduddin** [rafael-fuente/Diffraction-Simulations--Angular-Spectrum-Method#21] [Q&A] [answered]
I have two questions about visualization which are:

1. I would like to get full longitudinal profile plot for a big lens (in 60 cm wide)  but the `plot_longitudinal_profile_colors` function trimmed it on y axis. How could I deal with this kind of issue?
2. Is there any code available to create an animation of intensity plots?

Thanks, 
Irfan

**Comment by rafael-fuente** [comment, marked as answer]
Hi irfanimaduddin,

1. Implementing plot_longitudinal along the y axis is on the to-do list of things
2. Regarding the animations, I made them by iterating each frame separately and then merging into a single video using ffmpeg
